test/cases/azure.sh

```diff
@@ -201,7 +201,7 @@ if [[ "$CI_PROJECT_NAME" =~ "cloud-image-val" ]]; then
   TAG=${CI_COMMIT_REF_SLUG}
 else
   # If not, get prod container
-  TAG="prod"
+  TAG="pr-282" # Testing container from https://github.com/osbuild/cloud-image-val/pull/282
```

Comment:
TODO: Undo this change once CI passes.

Reply:
The azure.sh and azure.sh_hyperv_gen2 jobs are passing.
HyperV generation is correctly set for the deployed VMs.
CIV PR has been merged, so I will revert this container tag back to "prod".